shell/platform/embedder/tests/embedder_test_backingstore_producer.cc

```diff
@@ -258,6 +259,7 @@ bool EmbedderTestBackingStoreProducer::CreateSurface(
   backing_store_out->open_gl.surface.clear_current_callback = clear_current;
   backing_store_out->open_gl.surface.destruction_callback =
       destruction_callback;
+  backing_store_out->open_gl.surface.format = GL_RGBA8;
```

Comment:
Should this be `GL_BGRA8_EXT`? That would better match the previous test behavior. Or does that cause tests to fail on your machine?

Reply:
I have updated it to `GL_RGBA8_EXT` and the tests still pass on my machine. That being said, I wasn't sure what's the most suitable GL header (which contains this value) to be included in these tests, so I just copied the value instead.

I'm happy to replace the hard-coded value with a proper import if someone can point me to the correct header file that should be used.